charts/dispatch/charts/event-manager/templates/deployment.yaml

```diff
@@ -39,9 +39,10 @@ spec:
             - "--function-manager={{ .Release.Name }}-function-manager"
             - "--secret-store={{ .Release.Name }}-secret-store"
             # TODO: Read password from secret
-            - "--amqpurl=amqp://user:serverless@{{ .Release.Name }}-rabbitmq:5672/"
+            - "--rabbitmq-url=amqp://user:serverless@{{ .Release.Name }}-rabbitmq:5672/"
```

Comment:
not really, this is more to standardize the attribute naming (it was rabbitmq in some places and amqp in others). Note that we're using amqp library for 0.9.1 version with extensions for RabbitMQ. Although technically possible, I don't think it would be feasible to support other AMQP implementations than RabbitMQ.
 
Also, the plan is to add Kafka support, so there will be another flag for that too.